samples/unit-tests/series-area/stacked-area-threshold/demo.js

```diff
@@ -0,0 +1,107 @@
+function checkY(chart, assert) {
```

Comment:
Polluting the global namespace. Wrap the contents of this file in an iife or a QUnit.module.